vue/src/vue-elements/accounts-tab-panel.vue

```diff
@@ -1,55 +1,99 @@
 <template>
-    <div class="tab-view">
-        <div class="panel-body">
-            <div class="toast  toast-warning" v-html="labels.twitter_warning" v-if="twitter_warning">
-
+  <div class="tab-view">
+    <div class="panel-body">
+      <div
+        v-if="twitter_warning"
+        class="toast  toast-warning"
+        v-html="labels.twitter_warning"
+      />
+      <div class="container">
+        <div
+          class="columns"
+          :class="'rop-tab-state-'+is_loading"
+        >
+          <div class="column col-sm-12 col-md-12 col-lg-12 text-left rop-available-accounts mt-2">
+            <vue_spinner
+              v-if="is_preloading === 0"
+              ref="Preloader"
+              :preloader_message="labels.preloader_message_accounts"
+            />
+            <div
+              v-if="accountsCount === 0 && is_preloading > 0"
+              class="empty mb-2"
+            >
+              <div class="empty-icon">
+                <i class="fa fa-3x fa-user-circle-o" />
+              </div>
+              <p class="empty-title h5">
+                {{ labels.no_accounts }}
+              </p>
+              <p class="empty-subtitle">
+                {{ labels.no_accounts_desc }}
+              </p>
             </div>
-            <div class="container">
-                <div class="columns" :class="'rop-tab-state-'+is_loading">
-                    <div class="column col-sm-12 col-md-12 col-lg-12 text-left rop-available-accounts mt-2">
-                        <vue_spinner :preloader_message="labels.preloader_message_accounts" ref="Preloader" v-if="is_preloading === 0"></vue_spinner>
-                        <div class="empty mb-2" v-if="accountsCount === 0 && is_preloading > 0">
-                            <div class="empty-icon">
-                                <i class="fa fa-3x fa-user-circle-o"></i>
-                            </div>
-                            <p class="empty-title h5">{{labels.no_accounts}}</p>
-                            <p class="empty-subtitle">{{labels.no_accounts_desc}}</p>
-                        </div>
-                        <div class="account-container" v-for="( account, id ) in accounts" v-if="is_preloading > 0">
-                            <service-user-tile :account_data="account" :account_id="id"></service-user-tile>
-                            <span class="divider"></span>
-                        </div>
-                        <div class="add-accounts" v-if="is_preloading > 0" id="rop-add-account-button">
-                            <add-account-tile></add-account-tile>
-                            <span class="divider"></span>
-                        </div>
-                    </div>
-                </div>
+            <div
+              v-for="( account, id ) in accounts"
+              v-if="is_preloading > 0"
+              class="account-container"
+            >
```

Comment:
You can wrap this and apply the `if` condition on the wrapper as to not check for each loop item. Also the loop is missing a `:key`